addons/website_event_booth/views/event_type_views.xml

```diff
@@ -8,8 +8,8 @@
         <field name="arch" type="xml">
             <xpath expr="//span[@name='website_menu']" position='after'>
                 <span>
-                    <label for="booth_menu" string="Booth Menu Item"/>
-                    <field name="booth_menu"/>
+                    <field class="ms-5" name="booth_menu"/>
+                    <label for="booth_menu" string="Booths Menu Item"/>
```

Comment:
Umm, as mentioned in the task chatter by SHBH, we have to keep unique strings to avoid duplicate label errors like [these](https://runbot.odoo.com/runbot/batch/1458093/build/62529457).

Hence, I did not remove "Menu Item" from `event_type_views` since most of the strings would otherwise clash with the ones in `event_event_views`.